actionmailbox/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Rails 6.0.0.beta1 (January 18, 2019) ##
 
+*   Allow skipping incineration of processed emails.
+
+    This can be done by setting `config.action_mailbox.incinerate` to `false`.
+
+    *Pratik Naik*
+
```

Comment:
Last thing: this should go above the beta1 heading. beta1 already shipped.

Reply:
Ah, missed that. Pushed a fix.